### PR TITLE
[BG-161] Dodać defaultowe wartości dla algorytmów

### PR DESCRIPTION
**Pull request by Camel3333**

### CODE REVIEW

File: src/main/java/com/example/controller/settings/KingSettingsController.java [hunk 1]
```diff
@@ -46,7 +46,8 @@ public List<Node> getAllNodes() {
 
     @Override
     public void adjustSettingsConditions(MyGraph<Integer, Integer> graph) {
-        // intentionally do nothing - settings do not depend on graph
+        AlgorithmSetting<Integer> phaseSetting = (AlgorithmSetting<Integer>) algorithmSettings.getSettings().get("phase");
+        phaseSetting.setValue(graph.numVertices());
```
Comment (mkroczek) [inline src/main/java/com/example/controller/settings/KingSettingsController.java:50]: Proposed phase should be counted the same way as allowedNumberOfTraitors() in KingAlgorithm

File: src/main/java/com/example/controller/settings/LamportSettingsController.java [hunk 2]
```diff
@@ -50,6 +50,8 @@ public void adjustSettingsConditions(MyGraph<Integer, Integer> graph) {
         Vertex<Integer> commander = graph.vertices().stream().toList().get(0);
         int maxDepth = graph.getLongestPathFor(commander);
         algorithmSettings.getSettings().get("depth").setValidateArgument((value) -> (Integer) value >= 0 && (Integer) value <= maxDepth);
+        AlgorithmSetting<Integer> depthSetting = (AlgorithmSetting<Integer>) algorithmSettings.getSettings().get("depth");
+        depthSetting.setValue(maxDepth);
```
Comment (mkroczek) [inline src/main/java/com/example/controller/settings/LamportSettingsController.java:54]: I am not sure, but proposed depth should be equal to the number of traitors. Please ask Agnieszka to clarify it.

**Review by mkroczek — CHANGES_REQUESTED** [review]
Please fix condition in King Algorithm

File: src/main/java/com/example/controller/settings/LamportSettingsController.java (hunk 2 above)
Comment (syklzz) [reply src/main/java/com/example/controller/settings/LamportSettingsController.java:54]: It should be min value of traitors and max depth.

**Review by mkroczek — APPROVED** [review]

**Pull request merged**